.eslintrc.js

```diff
@@ -1,10 +1,110 @@
 module.exports = {
+	root: true,
+	globals: {
+		__DEV__: true,
+		process: true,
+		window: true,
+		document: true
+	},
 	env: {
 		node: true
 	},
-	extends: ['enact', 'plugin:prettier/recommended', 'prettier'],
-	plugins: ['import'],
+	extends: ['eslint:recommended', 'plugin:prettier/recommended', 'prettier'],
+	parser: '@babel/eslint-parser',
+	parserOptions: {
+		ecmaFeatures: {
+			jsx: true
+		},
+		ecmaVersion: 'latest',
+		sourceType: 'module',
+		requireConfigFile: false
+	},
+	plugins: ['@babel', 'import'],
 	rules: {
+		'block-scoped-var': 'warn',
+		curly: ['warn', 'multi-line'],
+		eqeqeq: ['warn', 'smart'],
```

Comment:
```suggestion
		'curly': ['warn', 'multi-line'],
		'eqeqeq': ['warn', 'smart'],
```

Reply:
Those 2 lines occurred lint errors

> /home/taeyoung.hong/webos_build_server/WRO-697/babel-preset-enact/.eslintrc.js
>   24:3  error  Replace `'curly'` with `curly`    prettier/prettier
>   25:3  error  Replace `'eqeqeq'` with `eqeqeq`  prettier/prettier
 
I will further analyze why those new errors occurred.

Reply:
These changes is due to a lint error detected in the prettier/prettier.
The original in eslint-config-enact is not detected because there is no prettier, but it is detected in the babel-preset-enact.
There are three ways to solve this problem.

1. Exclude .eslintrc.js from lint
2. Remove the prettier
3. Apply changes

Reply:
Let's change the option for prettier.